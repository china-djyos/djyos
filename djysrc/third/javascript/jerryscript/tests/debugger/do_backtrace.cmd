b do_backtrace.js:32
c
bt
c
