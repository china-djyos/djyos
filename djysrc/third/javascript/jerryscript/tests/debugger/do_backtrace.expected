Connecting to: localhost:5001
Stopped at tests/debugger/do_backtrace.js:15
(jerry-debugger) Breakpoint 1 at tests/debugger/do_backtrace.js:32 (in f4)
(jerry-debugger) Stopped at breakpoint:1 tests/debugger/do_backtrace.js:32 (in f4)
(jerry-debugger) Frame 0: tests/debugger/do_backtrace.js:32 (in f4)
Frame 1: tests/debugger/do_backtrace.js:39 (in foo)
Frame 2: tests/debugger/do_backtrace.js:48 (in test)
Frame 3: tests/debugger/do_backtrace.js:60
(jerry-debugger) Connection closed.
